### PR TITLE
hyperlinks and table formatting for supported boards

### PR DESCRIPTION
**Pull request by adamfeuer**
## Description

- hyperlinks and table formatting for supported boards table
- formatting changes for detailed support page that support hyperlinks

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- visual inspection
- manually testing links
- examining sphinx output for broken links

## Checklist:

- [x] documentation only changes

### CODE REVIEW

**Comment by adamfeuer** [comment]
@v01d Let me know if you want me to squash this into a single commit.

File: Documentation/introduction/detailed_support.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. todo::
+
   Migration is not yet completed from previous documentation. You can find the original list here:
```
Comment (v01d) [inline Documentation/introduction/detailed_support.rst:2]: this can be removed right? I understand this document is now completely migrated

Comment (adamfeuer) [reply Documentation/introduction/detailed_support.rst:2]: @v01d Oops. I should have removed it. I will push a change that does.

**Comment by v01d** [comment]
Thanks for that, must have been painful to check all those links. 
Could  you also remove all the ``|br|``? I will use CSS to make table elements vertically aligned (I understand that's why they're there). I recently also fixed wrapping off wide tables so line breaks are not needed to make the table fit either.

And yes, if you could squash after these changes, I'd appreciate it.

**Comment by adamfeuer** [comment]
@v01d Ok, I'll remove the `|br|`s – I'm glad you have the energy to fix the CSS, I didn't. 🙂 

The line breaks are there to wrap long lines... are we good with 120 character lines...? If so I'll do that too.

Re: painful to check all the links, yeah, but it was also kind of soothing. Sphinx checks all the links for you, so once I had all the links in the table fixed, all I had to do was make Sphinx happy. I was able to get the table links pretty quickly with some vim search and replace with regexes.

**Comment by v01d** [comment]
> @v01d Ok, I'll remove the `|br|`s – I'm glad you have the energy to fix the CSS, I didn't. slightly_smiling_face

Thanks!
 
> The line breaks are there to wrap long lines... are we good with 120 character lines...? If so I'll do that too.

Ah, I meant the HTML line breaks.

**Comment by adamfeuer** [comment]
@v01d Ok, cool– the only `|br|`s were to make the columns line up. I'll squash next and force push.

**Comment by adamfeuer** [comment]
@v01d I squashed and force pushed. You should be good to merge now.

**Comment by v01d** [comment]
Great, thanks!

**Pull request merged**